.env.demo

```diff
@@ -66,7 +66,7 @@ LAYMAN_CLIENT_URL=http://layman_client:3000/client/
 
 # client
 LAYMAN_CLIENT_PUBLIC_URL=http://localhost/client/
-LAYMAN_CLIENT_VERSION=v2.0.0
```

Comment:
Notification in CHANGELOG.md is missing. See for example commit [fe1b90e037a2313f61744c5dc6158f2910a91fde](https://github.com/LayerManager/layman/commit/fe1b90e037a2313f61744c5dc6158f2910a91fde)